components/base/CarouselCardList.vue

```diff
@@ -71,9 +71,10 @@
 
 <script lang="ts">
 import { Component, mixins, Prop } from 'nuxt-property-decorator'
-// import Identicon from '@polkadot/vue-identicon'
 import AuthMixin from '@/utils/mixins/authMixin'
 
+import type { CarouselNFTs } from './types'
```

Comment:
🙅‍♂️ never ever use `s`. With a combination with array I see in the next line is a good combo for the Bug :)